cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/LogAppenderInitializer.java

```diff
@@ -87,4 +98,15 @@ public void close() {
       LOG.info("Done stopping log appender {}", logAppender.getName());
     }
   }
+
+  private LoggerContext getLoggerContext() {
+    ILoggerFactory loggerFactory = LoggerFactory.getILoggerFactory();
+    // TODO: fix logging issue in mapreduce:  ENG-3279
```

Comment:
You can safely remove that. TODO. I don't think there is any logging issue in MR right now that need fixing (even there is, I doubt there is anything to do with the ENG-3279).